assignments/subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb/assign.tagging.json

```diff
@@ -0,0 +1,31 @@
+{
+    "properties": {
+      "metadata": {
+        "assignedBy": "PlatOps"
+      },
+      "description": "Policy to check required tags",
+      "displayName": "HMCTS Tagging - CFT SBOX",
+      "enforcementMode": "DoNotEnforce",
```

Comment:
Once we finish the tagging work we will switch "DoNotEnforce" to "Default" enforcementMode